README.rst

```diff
@@ -10,6 +10,9 @@ Django Graph API |travis| |slack| |rtd|
 .. |rtd| image:: https://readthedocs.org/projects/django-graph-api/badge/?version=latest
    :alt: Docs status on readthedocs
    :target: http://django-graph-api.readthedocs.io/
+.. |versions| image:: https://img.shields.io/pypi/pyversions/django-graph-api.svg
```

Comment:
This shield doesn't seem to be showing up in your repo's readme. I'm not sure why.